stdlib/ext/dist-ext.ext-ocaml.mc

```diff
@@ -0,0 +1,59 @@
+
+
+include "map.mc"
+include "ocaml/ast.mc"
+
+let distExtMap =
+  use OCamlTypeAst in
+  mapFromSeq cmpString
+  [
```

Comment:
The external implementation record has been extended with a field `cLibraries` to allow specifying c libraries that are needed during linking. I.e. the type has been changed to
```
type ExternalImpl = {
  ident : String,
  ty : Type,
  libraries : [String],
  cLibraries : [String]
}
```
so you should add a field `cLibraries = []` to all of these.

Reply:
Thanks. Fixed.